contracts/ToucanRegenBridge.sol

```diff
@@ -180,6 +180,7 @@ contract ToucanRegenBridge is Ownable, Pausable, AccessControl {
         require(!origins[origin], "duplicate origin");
         origins[origin] = true;
 
+        totalTransferred -= amount;
```

Comment:
Actually I think we can move this after `tco2Limits[tco2] -= amount;` and have it run `unchecked` since if `tco2Limits[tco2] -= amount;` does not underflow then it's not possible for `totalTransferred -= amount;` to underflow either:
```solidity
unchecked {
  // not possible to underflow if tco2Limits[tco2] above
  // does not underflow
  totalTransferred -= amount;
}
```

Reply:
We can also apply a similar optimization in `bridge` but have `tco2Limits[tco2] += amount;` be `unchecked` since if `totalTransferred += amount;` does not overflow then it's impossible for `tco2Limits[tco2] += amount;` to overflow.